schematic/store/synapse.py

```diff
@@ -1975,62 +2065,56 @@ def createTable(synStore: SynapseStorage, tableToLoad: pd.DataFrame = None, tabl
             # column types.
-            table = build_table(tableName, datasetParentProject, tableToLoad)
-            table = synStore.syn.store(table, isRestricted = restrict)
+            table = build_table(self.tableName, datasetParentProject, self.tableToLoad)
+            table = self.synStore.syn.store(table, isRestricted = self.restrict)
             return table.schema.id
 
-    def replaceTable(synStore: SynapseStorage, tableToLoad: pd.DataFrame = None, tableName: str = None, existingTableId: str = None, specifySchema: bool = True, datasetId: str = None, columnTypeDict: dict = None, restrict: bool = False):
+    def replaceTable(self, specifySchema: bool = True, columnTypeDict: dict = None,):
         """
         Method to replace an existing table on synapse with metadata from a new manifest
         
         Args:
-            tableToLoad: manifest formatted appropriately for the table
-            tableName: name of the table to be uploaded
-            existingTableId: synId of the existing table to be replaced
             specifySchema: to infer a schema for the table format      
-            datasetId: synID of the dataset for the manifest    
-            columnTypeDict: dictionary schema for table columns: type, size, etc
-            restrict: bool, whether or not the manifest contains sensitive data that will need additional access restrictions 
-            
+            columnTypeDict: dictionary schema for table columns: type, size, etc            
 
         Returns:
            existingTableId: synID of the already existing table that had its metadata replaced
         """
-        datasetEntity = synStore.syn.get(datasetId, downloadFile = False)
+        datasetEntity = self.synStore.syn.get(self.datasetId, downloadFile = False)
         datasetName = datasetEntity.name
-        table_schema_by_cname = synStore._get_table_schema_by_cname(columnTypeDict) 
-        existing_table, existing_results = synStore.get_synapse_table(existingTableId)
+        table_schema_by_cname = self.synStore._get_table_schema_by_cname(columnTypeDict) 
+        existing_table, existing_results = self.synStore.get_synapse_table(self.existingTableId)
         # remove rows
-        synStore.syn.delete(existing_results)
+        self.synStore.syn.delete(existing_results)
         # wait for row deletion to finish on synapse before getting empty table
         sleep(10)
         
         # removes all current columns
-        current_table = synStore.syn.get(existingTableId)
-        current_columns = synStore.syn.getTableColumns(current_table)
+        current_table = self.synStore.syn.get(self.existingTableId)
+        current_columns = self.synStore.syn.getTableColumns(current_table)
         for col in current_columns:
             current_table.removeColumn(col)
 
-        if not tableName:
-            tableName = datasetName + 'table'
+        if not self.tableName:
```

Comment:
Nit: `self.tableName` is default value is None and its type is `str`. I recommend to be specific

```
if self.tableName is not None
```

Not that this will happen, but if you set the value as `0`. you will get something like this:

```
tableName = 0
datasetName = "foo"
if not tableName:
    tableName = datasetName + 'table'
print(tableName)
# footable
```